cg/cli/downsample.py

```diff
@@ -0,0 +1,81 @@
+"""cg module for downsampling reads in a sample."""
+
+import logging
+from typing import List
+
+import click
+
+from cg.apps.downsample.downsample import DownSampleAPI
+from cg.apps.downsample.utils import format_downsample_case
+from cg.constants.constants import DRY_RUN
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def downsample_cmd():
+    """Downsample reads in a sample."""
+
+
+@downsample_cmd.command("samples", help="Downsample reads in one or multiple samples in a case.")
+@click.option(
+    "-c",
+    "--case-internal-id",
```

Comment:
do we want to rename this?

Reply:
At some point we tried had an effort to be more explicit in what "ids" one can refer to. 
case_id is ambiguous as it can refer to the `database_row` which is `case.id`. 
`case.internal_id` is the pet name. 

The same goes for `sample_id` and `sample_internal_id`